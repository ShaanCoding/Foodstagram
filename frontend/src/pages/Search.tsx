// import styles from "../styles/Search.module.css";


import { useParams } from 'react-router-dom'
import useAuth from '../api/util/useAuth'

const Search = () => {
  const [account, isLoading] = useAuth()
  const { searchStr } = useParams();
  console.log(searchStr)

  return(
    

    <div>
      
    </div>
	)
}

export default Search