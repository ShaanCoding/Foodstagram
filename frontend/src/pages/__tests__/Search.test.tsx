import { QueryClient, QueryClientProvider } from 'react-query'
import { BrowserRouter, Router } from 'react-router-dom'
import renderer from 'react-test-renderer'
import Search from '../Search'

const queryClient = new QueryClient()

it('The Search page renders correctly', () => {
	const tree = renderer
		.create(
			<QueryClientProvider client={queryClient}>
				<BrowserRouter>
					<Search />
				</BrowserRouter>
			</QueryClientProvider>
		)
		.toJSON()
	expect(tree).toMatchSnapshot()
})
