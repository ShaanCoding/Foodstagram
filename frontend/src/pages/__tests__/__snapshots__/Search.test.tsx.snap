// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Search page renders correctly 1`] = `<div />`;
